packages/middleware-location-constraint/src/index.ts

```diff
@@ -24,6 +24,7 @@ export function locationConstraintMiddleware(
     args: InitializeHandlerArguments<any>
   ): Promise<InitializeHandlerOutput<Output>> => {
     const { CreateBucketConfiguration } = args.input;
+    const region = await options.region();
```

Comment:
Can you add comment that the region will always be a `Provider<string>` after region config resolver?

Reply:
comment in 448d2f897f7c997af77c839eedaa2e301399b2ea